src/misc.jl

```diff
@@ -201,3 +201,55 @@ end
     root --> :top
     GraphPlot((source, destiny))
 end
+
+
+@userplot AndrewsPlot
+
+"""
+    andrewsplot(data::AbstractArray; kind=[])
+
+https://en.wikipedia.org/wiki/Andrews_plot
+
+Example
+```julia
+using RDatasets, PlotRecipes
+iris   = dataset("datasets", "iris")
+y      = Array(iris[:,1:4])
+kind   = Array(iris[:,5])
+andrewsplot(y, kind=kind)
+```
+"""
+andrewsplot
+
+@recipe function f(h::AndrewsPlot; kind=[])
+    y = h.args[1]
```

Comment:
I had to do like so
`y = convert(Array, DataArray(y), NaN)`
The `DataArray(y)` seems very cheap when `y` already is a `DataArray`, so it shouldn't be much of a problem.